custom-login/package.json

```diff
@@ -21,8 +21,8 @@
     "@angular/router": "~14.2.0",
     "@giomamaladze/ng2-semantic-ui": "^1.0.1",
     "@okta/okta-angular": "^6.0.0",
-    "@okta/okta-auth-js": "7.0.2",
-    "@okta/okta-signin-widget": "7.14.0",
+    "@okta/okta-auth-js": "7.9.0",
+    "@okta/okta-signin-widget": "7.26.1",
```

Comment:
These are exact version pins. Again, intentional decision? Something special to Angular?